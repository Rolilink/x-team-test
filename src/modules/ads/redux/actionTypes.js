export const ADD_AD = 'x-team/discount-ascii-warehouse/ads/ADD';
export const CLEAR_ADS = 'x-team/discount-ascii-warehouse/ads/CLEAR';
