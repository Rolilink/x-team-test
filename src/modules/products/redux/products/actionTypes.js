import { PENDING, FULFILLED, REJECTED } from 'redux-promise-middleware';

export const FETCH_PRODUCTS = 'x-team/discount-ascii-warehouse/products/FETCH';
export const FETCH_PRODUCTS_SUCCESS = `${FETCH_PRODUCTS}_${FULFILLED}`;
export const FETCH_PRODUCTS_ERROR = `${FETCH_PRODUCTS}_${REJECTED}`;
export const FETCH_PRODUCTS_LOADING = `${FETCH_PRODUCTS}_${PENDING}`;
