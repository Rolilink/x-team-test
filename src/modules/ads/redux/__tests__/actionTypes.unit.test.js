import { ADD_AD, CLEAR_ADS } from '../actionTypes';

describe('Ads.Redux.ActionTypes', () => {
  it('should contain ADD_AD actionType', () => expect(ADD_AD).toMatchSnapshot());
  it('should contain CLEAR_ADS actionType', () => expect(CLEAR_ADS).toMatchSnapshot());
});
