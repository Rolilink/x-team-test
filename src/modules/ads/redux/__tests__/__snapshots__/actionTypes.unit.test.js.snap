// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ads.Redux.ActionTypes should contain ADD_AD actionType 1`] = `"x-team/discount-ascii-warehouse/ads/ADD"`;

exports[`Ads.Redux.ActionTypes should contain CLEAR_ADS actionType 1`] = `"x-team/discount-ascii-warehouse/ads/CLEAR"`;
